test-control/Test/Control/Concurrent/Class/MonadSTM/RWVar.hs

```diff
@@ -17,44 +19,71 @@ tests = testGroup "Test.Control.Concurrent.Class.MonadSTM.RWVar" [
 data Action a = Read a | Incr a
```

Comment:
Any particular reason why this is polymorphic on `a` if we use it with `()` later on?

Reply:
It doubles as the result of reading `say` messages as well, in which case it holds `Int`